sci-biology/emboss/emboss-6.6.0.ebuild

```diff
@@ -1,28 +1,19 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Header: $
 
-EAPI="4"
+EAPI=5
 
-inherit emboss eutils
-
-#EBO_PATCH="4"
-#EBOV=${PV/_p*}
+inherit autotools-utils emboss eutils
 
 EBO_PATCH=""
 EBOV=${PV}
+EBO_EAUTORECONF=yes
 
 DESCRIPTION="The European Molecular Biology Open Software Suite - A sequence analysis package"
-SRC_URI="
-	ftp://emboss.open-bio.org/pub/EMBOSS/EMBOSS-${PV}.tar.gz"
-# ftp://emboss.open-bio.org/pub/EMBOSS/EMBOSS-6.6.0.tar.gz
-# ftp://emboss.open-bio.org/pub/EMBOSS/old/6.5.0/EMBOSS-6.5.7.tar.gz
-
-##[[ -n ${EBO_PATCH} ]] && SRC_URI+=" ftp://${PN}.open-bio.org/pub/EMBOSS/fixes/patches/patch-1-${EBO_PATCH}.gz -> ${P}-upstream.patch.gz"
-#[[ -n ${EBO_PATCH} ]] && SRC_URI+=" http://pkgs.fedoraproject.org/lookaside/pkgs/EMBOSS/patch-1-4.gz/7a42594c5eda4adc6457f33e4ab0d8f2/patch-1-${EBO_PATCH}.gz -> ${P}-upstream.patch.gz"
+SRC_URI="ftp://emboss.open-bio.org/pub/EMBOSS/EMBOSS-${PV}.tar.gz"
 
-#KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
-KEYWORDS=""
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
```

Comment:
Are all bugs fixed in this version? We had massive bug reports, because of which the keywords are dropped.